src/plugins/intel_cpu/src/mkldnn_plugin.cpp

```diff
@@ -739,20 +807,14 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
         capabilities.push_back(METRIC_VALUE(FP16));
         capabilities.push_back(METRIC_VALUE(INT8));
         capabilities.push_back(METRIC_VALUE(BIN));
-        IE_SET_METRIC_RETURN(OPTIMIZATION_CAPABILITIES, capabilities);
-    } else if (name == METRIC_KEY(SUPPORTED_CONFIG_KEYS)) {
-        std::vector<std::string> configKeys;
-        for (auto && opt : engConfig._config)
-            configKeys.push_back(opt.first);
-        IE_SET_METRIC_RETURN(SUPPORTED_CONFIG_KEYS, configKeys);
-    } else if (name == METRIC_KEY(RANGE_FOR_ASYNC_INFER_REQUESTS)) {
-        std::tuple<unsigned int, unsigned int, unsigned int> range = std::make_tuple(1, 1, 1);
-        IE_SET_METRIC_RETURN(RANGE_FOR_ASYNC_INFER_REQUESTS, range);
-    } else if (name == METRIC_KEY(RANGE_FOR_STREAMS)) {
-        std::tuple<unsigned int, unsigned int> range = std::make_tuple(1, parallel_get_max_threads());
-        IE_SET_METRIC_RETURN(RANGE_FOR_STREAMS, range);
-    } else if (name == METRIC_KEY(IMPORT_EXPORT_SUPPORT)) {
-        IE_SET_METRIC_RETURN(IMPORT_EXPORT_SUPPORT, true);
+        capabilities.push_back("IMPORT_EXPORT");
+        return capabilities;
+    } else if (name == ov::range_for_async_infer_requests) {
+        const std::tuple<unsigned int, unsigned int, unsigned int> range = std::make_tuple(1, 1, 1);
+        return range;
+    } else if (name == ov::range_for_streams) {
+        const std::tuple<unsigned int, unsigned int> range = std::make_tuple(1, parallel_get_max_threads());
+        return range;
     } else {
         IE_THROW() << "Unsupported metric key " << name;
     }
```

Comment:
should we also update SetConfig logic to support new RW properties? `ov::streams::num`, `ov::hint::inference_precision` and maybe others

Reply:
Yes, we should
Wanted to discuss, how we should map new and legacy options in case of renaming.
For example in case of ov::streams::num I think it would be better to update ie_streams_executor in order to use new names and implement mapping inside SetConfig() so when we finally remove legacy configuration we would just remove mapping.
But this will affect other plugins (like TEMPLATE), so should be done as single commit.

Other option is to map new properties to legacy ones, but later it will be harder to drop legacy configuration.

Reply:
@dmitry-gorokhov FYI, implemented as second option